base/_mixins.scss

```diff
@@ -222,8 +222,32 @@
 // @jhogue
 @mixin javascript-listener( $content ) {
 	body:after {
-		content: $content; 
-		display: none; 
-		speak: none; 
+		content: $content;
+		display: none;
+		speak: none;
+	}
+}
+
+// Icon Font Mixin
+// @accepts string(s)
+// @param: - $icon - "\f105"
+// @param: - $positin - before or after
+// @param: - $color - any hex value
+// @usage: icon("\f105", before, palette(brand, primary));
+@mixin icon($icon, $position:before, $color:inherit) {
+	&::#{$position} {
+		content: $icon;
+		color: $color;
+		display: inline-block;
+		font-family: 'Custom Font Family';
+		font-style: normal;
+		font-weight: normal;
+		text-decoration: inherit;
+		margin-right: .5em;
+		-webkit-font-smoothing: antialiased;
+		moz-osx-font-smoothing: grayscale;
+		@if $position == after {
+			margin-left: .5em;
+		}
```

Comment:
So `after`s get margins on both sides, but `before`s only get it on the right? 😛